packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -179,6 +182,11 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
         symbol: 'MATIC',
         explorer: 'https://polygonscan.com/',
       },
+      [KnownChainIds.GnosisMainnet]: {
```

Comment:
Add to `evmChainIds` and `EvmChainAdapter`

Reply:
I see there is no chain adapter added yet. I would  either add that now, or bring the chain adapters changes into a follow up PR and do the full package

Reply:
I think I've left the adapter out as it depends on the api impl which isn't there yet, but will double check

Reply:
Yeah correct, left it for the api PR